pkg/stub/handler.go

```diff
@@ -117,7 +128,13 @@ func (h *Handler) syncIngressUpdate(ci *ingressv1alpha1.ClusterIngress) error {
 			err = sdk.Create(service)
 			if err == nil {
 				logrus.Infof("created router service %s/%s", service.Namespace, service.Name)
-			} else if !errors.IsAlreadyExists(err) {
+			} else if errors.IsAlreadyExists(err) {
+				err = sdk.Update(service)
```

Comment:
I don't think we need to update service, none of the spec changes in CR will change service object.

Reply:
you're right, removing this code